src/pages/SignUp/SignUp.jsx

```diff
@@ -1,38 +1,63 @@
+import { useState } from "react";
+import { useDispatch } from "react-redux";
 import { Link } from "react-router-dom";
+import { signUpUser } from "../../store/authSlice";
+import { useNavigate } from "react-router-dom";
 import "./SignUp.css";
 
 const SignUp = () => {
+  const [firstName, setFirstName] = useState("");
+  const [lastName, setLastName] = useState("");
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
```

Comment:
this can be combined into one usestate of object like useState({firstname:"", lastname:""}) so on

Reply:
will do later